dart/lib/src/sentry_exception_factory.dart

```diff
@@ -0,0 +1,33 @@
+import 'protocol/sentry_stack_trace.dart';
+import 'protocol.dart';
+import 'sentry_stack_trace_factory.dart';
+
+/// class to convert Dart Error and exception to SentryException
+class SentryExceptionFactory {
+  final SentryStackTraceFactory _stacktraceFactory;
+
+  const SentryExceptionFactory({
+    SentryStackTraceFactory stacktraceFactory = const SentryStackTraceFactory(),
+  }) : _stacktraceFactory = stacktraceFactory;
+
+  SentryException getSentryException(
+    dynamic exception, {
+    StackTrace stackTrace,
```

Comment:
`getStackFrames(dynamic stackTrace)` is a `dynamic`, but here a `StackTrace`, as Dart fully allows a `StackTrace` to be even a String, we should probably use only `dynamic`

Reply:
✅